pkg/performanceprofile/controller/performanceprofile/components/tuned/tuned.go

```diff
@@ -214,13 +214,22 @@ func NewNodePerformance(profile *performancev2.PerformanceProfile) (*tunedv1.Tun
 	if err != nil {
 		return nil, err
 	}
-
+	RealTimeKernelProfileData, err := getProfileData(filepath.Join("tuned", components.ProfileNamePerformanceRT), templateArgs)
+	if err != nil {
+		return nil, err
+	}
 	name := components.GetComponentName(profile.Name, components.ProfileNamePerformance)
+	RealTimeKernelProfileName := components.ProfileNamePerformanceRT
```

Comment:
You need to use the GetComponentName method here too, otherwise two perf profiles will try to create the same profile.

Reply:
I don't get why two perf profiles will try to create the same profile.
Anyway I tried to do that before, what happend was that the tuned profile got posted to the cluster with the name of the profile, i. e "-performance"  suffix (openshift-node-performance-rt-performance) while the include section is looking for hardcoded "openshift-node-performance-rt"
`2024-02-14 10:49:19,013 ERROR    tuned.daemon.daemon: Cannot set initial profile. No tunings will be enabled: Cannot load profile(s) 'openshift-node-performance-performance': Cannot find profile 'openshift-node-performance-rt' in '['/etc/tuned', '/usr/lib/tuned']'.
`

Reply:
Think about the flow when you have two PerformanceProfiles in a cluster:

Each performance profile generates a new Tuned object that contains the tuned profiles. So two performance profiles result in two Tuned objects that will both contain the same openshift-node-performance-rt tuned profile. This results in NTO trying to save both of the defined profiles into the same filename.

You are right the include will have to be improved to refer to the proper name. Template variable with the perf profile name can do that.

Reply:
Fixed that.